### DISCUSSION TITLE
Why JCB 5.0.3 is not working properly on XAMPP?

### DISCUSSION THREAD

**Discussion by morphinestyle** [joomengine/Joomla-Component-Builder#977] [Q&A]
I read that someone has already had problems with XAMPP: the component does not load data from the database:

![image](https://github.com/user-attachments/assets/3934f94b-af9f-43bf-97fd-4f1b70cca09a)

This is the Joomla!/XAMPP configuration:

![image](https://github.com/user-attachments/assets/1b861889-1bf1-4d9f-bb62-1bbf9b69d3a8)

Has anyone solved this yet?

**Comment by Llewellynvdm** [comment]
You must please move issues to https://git.vdm.dev/joomla/Component-Builder/issues since we ill not respond to issues on the discussion form.
